### PR TITLE
Update Kodi to 19.1 and re-add pycryptodome and python-pillow modules (needed by some addons)

### PR DESCRIPTION
**Pull request by gipawu**

### CODE REVIEW

File: tv.kodi.Kodi.metainfo.xml [hunk 1]
```diff
@@ -32,14 +32,10 @@
     </description>
     <launchable type="desktop-id">tv.kodi.Kodi.desktop</launchable>
     <screenshots>
-        <screenshot type="default">https://kodi.tv/sites/default/files/styles/hero_16_9/public/feature/field_image/2017-02-16_00-38-09.png</screenshot>
-        <screenshot type="default">https://kodi.tv/sites/default/files/styles/hero_16_9/public/feature/field_image/about--movies-list.jpg</screenshot>
-        <screenshot type="default">https://kodi.tv/sites/default/files/styles/hero_16_9/public/feature/field_image/2017-02-16_00-46-13.jpg</screenshot>
-        <screenshot type="default">https://kodi.tv/sites/default/files/styles/hero_16_9/public/feature/field_image/2017-02-16_00-34-04.png</screenshot>
-        <screenshot type="default">https://kodi.tv/sites/default/files/styles/hero_16_9/public/feature/field_image/about--pvr.jpg</screenshot>
-        <screenshot type="default">https://kodi.tv/sites/default/files/styles/hero_16_9/public/feature/field_image/2017-02-16_00-54-04.png</screenshot>
+        <screenshot type="default">https://cdn.dday.it/system/uploads/news/main_image/22334/main_a1.jpg</screenshot>
```
Comment (wsnipex) [inline tv.kodi.Kodi.metainfo.xml:35]: please use:
https://kodi.tv/images/skin-estuary-example.webp
https://kodi.tv/images/about--movies.webp
https://kodi.tv/images/about--tvshows.webp
https://kodi.tv/images/about--music.webp
https://kodi.tv/images/about--pvr.webp
https://kodi.tv/images/about--games.webp
https://kodi.tv/images/about--addons.webp

**Review by wsnipex — COMMENTED** [review]
Thank you!
Please squash the fixup commits

File: tv.kodi.Kodi.metainfo.xml (hunk 1 above)
Comment (gipawu) [reply tv.kodi.Kodi.metainfo.xml:35]: I tried to use webp screenshot, but test build failed (see test build 49125)

Comment (14mRh4X0r) [reply tv.kodi.Kodi.metainfo.xml:35]: The validator uses Gdk-PixBuf to load the image, which does not support WebP by default, even though [the spec says WebP is fine](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-screenshots). The Kodi website seems to exclusively use WebP for its images…

Comment (wsnipex) [reply tv.kodi.Kodi.metainfo.xml:35]: ok, lets keep it as is for now. Maybe we can get a jpeg screenshot up in the future.

**Comment by wsnipex** [comment]
thank you!

**Pull request merged**